src/controller/NoteController.ts

```diff
@@ -25,20 +25,24 @@ export class NoteController {
                         break;
                     case 'date':
 
-                        let first;
-                        let second;
-                        const dateOne = new Date(value[0]);
-                        const dateTwo = new Date(value[1]);
+                        if (Array.isArray(value)) {
+                            let first;
+                            let second;
+                            const dateOne = new Date(value[0]);
+                            const dateTwo = new Date(value[1]);
 
-                        if (dateOne < dateTwo) {
-                            first = startOfDay(dateOne);
-                            second = endOfDay(dateTwo);
+                            if (dateOne < dateTwo) {
+                                first = startOfDay(dateOne);
+                                second = endOfDay(dateTwo);
+                            } else {
+                                first = startOfDay(dateTwo);
+                                second = endOfDay(dateOne);
```

Comment:
nit - converting these `first` and `second` variables to consts instead of lets with a ternary statement in each?

Reply:
Cool cool just did that